source/databases-collections.txt

```diff
@@ -45,6 +45,10 @@ Click a collection to view its documents, schema, and indexes.
 Documents
 ``````````
 
+When you expand a collection, that collection's document count is 
+displayed next to the :guilabel:`Documents` item in the left navigation 
+panel.
```

Comment:
[nits]
- I think we can use more active voice
- Maybe `label` is a better word than `item`?
- JW brought up in another review that the navigation may not be on the left depending on how you have your VSCode set up.

```suggestion
When you expand a collection, |vsce| displays that collection's document count
next to the :guilabel:`Documents` label in the navigation 
panel.
```

Reply:
All agreed, good points